sources/packages/web/src/types/contracts/student/ApplicationContract.ts

```diff
@@ -139,3 +139,46 @@ export interface ApplicationSummaryDTO {
   award: string;
   status: string;
 }
+
+/**
+ * DTO for Notice of Assessment view
+ */
+export interface NoticeOfAssessmentDTO {
+  assessment: Assessment;
+  applicationNumber: string;
+  fullName: string;
+  programName: string;
+  locationName: string;
+  offeringStudyStartDate: string;
+  offeringStudyEndDate: string;
+  msfaaNumber: string;
+  disbursement: any;
+}
+
+/**
+ * DTO for assessment payload
```

Comment:
Can we add a comment on this DTO, here and in the API, to make it clear that although the properties are explicit, this represents a dynamic jsonB database value that could be dynamically expanded by the workflow adding more values that will also be available on Form definition to be consumed?

Reply:
An elaborated comment has been provided to this interface as discussed.